### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ArtofNFTs**
Jason Schuck - learning about NFTs w/ real estate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,4 +93,4 @@ Event website (MIAMI, December 2021): https://www.dcentralcon.com
 * William Penney III
 * Maciej Frik
 * Scott Andersen
-* ADD YOUR NAME ABOVE HERE
```
Comment (YanesIsHere) [inline README.md:96]: Danny Sanchez

File: README.md [hunk 2]
```diff
@@ -93,4 +93,4 @@ Event website (MIAMI, December 2021): https://www.dcentralcon.com
 * William Penney III
 * Maciej Frik
 * Scott Andersen
```
Comment (YanesIsHere) [inline README.md:95]: Dany Sanchez

Comment (fulldecent) [reply README.md:95]: You have added a comment on top of somebody else's pull request. Please start again and create a new (your own) pull request with your change.

**Review by fulldecent — CHANGES_REQUESTED** [review]
This pull request adds your name (good!) but also removes the instructions at bottom (bad!).

Please start over and try again or edit the file again in this pull request.

**Pull request closed without merging**